File: market-application/src/main/java/com/example/market/application/port/in/RecordInspectionArrivalUseCase.java
package com.example.market.application.port.in;

import com.example.market.application.command.RecordInspectionArrivalCommand;
import com.example.market.domain.inspection.InspectionRequest;

public interface RecordInspectionArrivalUseCase {
    InspectionRequest arrive(RecordInspectionArrivalCommand command);
}

File: market-application/src/main/kotlin/com/example/market/application/port/in/RecordInspectionArrivalUseCase.kt
package com.example.market.application.port.`in`

import com.example.market.application.command.RecordInspectionArrivalCommand
import com.example.market.domain.inspection.InspectionRequest

interface RecordInspectionArrivalUseCase {
    fun arrive(command: RecordInspectionArrivalCommand): InspectionRequest
}
